generatedTypes/src/incubator/TextField/ValidationMessage.d.ts

```diff
@@ -14,9 +15,10 @@ export interface ValidationMessageProps {
      */
     validationMessageStyle?: TextStyle;
     retainSpace?: boolean;
+    validate: FieldStateProps['validateOnChange'];
```

Comment:
```suggestion
    validate?: FieldStateProps['validateOnChange'];
```

Reply:
Somehow this `FieldStateProps['validateOnChange']` is a `boolean` value, and it creates typing errors in the demo-screen on `validate` prop, that getting there an array of strings

Reply:
done